packages/react/src/theme/default-theme.ts

```diff
@@ -44,6 +44,31 @@ export const DEFAULT_THEME_OPTIONS: RecursivePartial<Theme> = {
       },
     },
   },
+  components: {
+    MuiButton: {
+      styleOverrides: {
+        containedPrimary: {
+          background: `linear-gradient(270deg, var(--oxygen-palette-primary-gradient-stop2),
```

Comment:
These two variables are now available in the theme. Hence we can use them and remove the two vars in `button.scss`.
```
--oxygen-palette-gradients-primary-stop1
--oxygen-palette-gradients-primary-stop2
```
```suggestion
          background: `linear-gradient(270deg, var(--oxygen-palette-primary-gradient-stop2),
```